kairon/events/definitions/mail_channel.py

```
@@ -0,0 +1,98 @@
+        return MailProcessor.validate_imap_connection(self.bot)
+
+    def enqueue(self, **kwargs):
+        """
+        Send event to event server.
+        """
+        try:
+            payload = {'bot': self.bot, 'user': self.user}
+            Utility.request_event_server(EventClass.mail_channel_read_mails, payload)
+        except Exception as e:
+            logger.error(str(e))
+            raise AppException(e)
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)